config.go

```diff
@@ -42,8 +42,10 @@ type Config struct {
 
 // NewConfig returns a new configuration instance with sane defaults.
 func NewConfig() *Config {
+	config := *sarama.NewConfig()
+	config.Version = sarama.V0_9_0_0
```

Comment:
So I prefer to define a `const minVersion = sarama.V0_9_0_0` and assign `minVersion` here.